inference_rules.adoc

```diff
@@ -129,6 +129,21 @@ MLCommons shall retain a library of past audit reports and send copies to MLComm
 
 An audit is expected to be completed within a 90 day period. Audits failing to meet this timeline can be requested to be invalidated by the auditee. The final decision to accept such a request will be taken by the Working Group.
 
+=== Late Submissions and Late Upload of Logs
+
+To ensure fairness to all submitters and the integrity of the process, MLPerf Inference will strictly adhere to the https://github.com/mlcommons/policies/blob/master/submission_rules.adoc#late-submissions[MLCommons general submission rules]. The 60 minute grace period will be strictly enforced, and any late submissions will not be considered as valid submissions. Exceptions will be made only in extraordinary circumstances as listed in the https://github.com/mlcommons/policies/blob/master/submission_rules.adoc#post-submission-extension-for-extraordinary-circumstances-submission-deadline-72-hours[mlcommons guidelines].
```

Comment:
This PR is much appreciated considering the amount of time which is spent during the review meetings on such requests. The one concern though is even though this PR is making the rules strict it is not preventing any such late submission requests from coming to the review committee. Also we know that the rules are a bit relaxed for first time submitters - but "how relaxed" is not defined. It would be good if the rules are clear so that the review committee chair can show that to the submitters asking for allowing any late submissions instead of those requests coming to the review committee for discussions.